_pages/en_US/dumping-nand.md

```diff
@@ -5,26 +5,32 @@ redirect_from:
   - /dump-nand
 ---
 
-In this step, we'll be making a NAND backup. This is a 1:1 copy of your DSi internal storage, which we can use later on to setup hiyaCFW and NO$GBA.
+In this step, we'll be making a NAND backup. This is a 1:1 copy of your Nintendo DSi internal storage, which we can use later on to setup hiyaCFW and NO$GBA.
 
-We highly recommend that you do this. A NAND backup can be used as a restore point in the future, in case your DSi stops working.
+We highly recommend that you do this. A NAND backup can be used as a restore point in the future, in case your Nintendo DSi stops working.
 {: .notice--primary}
 
-## Instructions
+## Section I - SD card setup
+Using a Windows, Linux or Mac OS device? We have a tool that will automatically set up your SD card for following the guide.
+{: .notice--info}
 
 1. Download the latest release of [dumpTool](https://github.com/zoogie/dumpTool/releases)
 1. Place `dumpTool.nds` anywhere on your SD card
-1. Launch the Homebrew Launcher through your exploit
-  - If you have already installed Unlaunch, use Unlaunch's homebrew launcher instead by holding **(A)** + **(B)** on your system boot and selecting it in the menu
-1. Follow the on-screen prompts
-  - This usually takes around seven minutes
-1. Hit the **(START)** button on your Nintendo DSi to exit when it is completed
-1. Power off your console and insert your SD card back into your computer
+
+## Section II - Dumping the NAND
+1. Launch `dumpTool.nds` through TWiLight Menu++
+1. Hit the **(A)** button on your Nintendo DSi to start dumping your NAND
```

Comment:
Oh, this applies to all buttons, but just remembered now. Maybe do `<kbd>A</kbd>`, you might need to edit the CSS a bit if its not being used but with some decent CSS that can make a nice clear button indicator like on GitHub: <kbd>A</kbd>